source/extensions/filters/network/zookeeper_proxy/filter.h

```diff
@@ -90,13 +146,21 @@ struct ZooKeeperProxyStats {
   ALL_ZOOKEEPER_PROXY_STATS(GENERATE_COUNTER_STRUCT)
 };
 
+enum class ErrorBudgetResponseType { FAST, SLOW, NONE };
+
+using LatencyThresholdList = const Protobuf::RepeatedPtrField<
```

Comment:
remove const or `ConstLatencyThresholdList` but the former is preferred

Reply:
Thanks, just made the change to remove const from alias.